### DISCUSSION TITLE
Batch sensor film and sampler handling

### DISCUSSION THREAD

**Discussion by leroyvn** [mitsuba-renderer/mitsuba3#786] [Q&A] [answered]
Hi, 

I'm asking myself a few questions about the `batch` sensor:

* Should I care about the reconstruction filter I assign to each subsensor, or is the one of the `batch` sensor's film used?
* Should I care about the sampler I assign to each subsensor, or is the one of the `batch` sensor used?

**Comment by njroussel** [comment, marked as answer]
Hi @leroyvn 

This is a great question, I'll add my answer to the documentation of the `batch` plugin too.

The `batch`'s sampler and reconstruction filter supersede those of the underlying sensors. This is at least true for the few sensors & films that are shipped in Mitsuba. There is no mechanism to guarantee that a custom sensor would still somehow use its own sampler or film.

**Reply by leroyvn** [reply]
Thanks for this quick reply :pray:
